playground/mocks/data/idp/idx/user-unlock-account-identifier-first.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Yeah you're right actually. I don't fully remember how it ended up this way but I think since the backend handles automatically challenging the 1 available authenticator if it sees that only one is available, the mock files end up being the same. I might have added this since they're used in different spec test flows, but tbh it's probably fine to just use the `user-unlock-account-identifier-first.json` mock for both 👍  I'll update that!